stackoverflow/src/main/java/ro/utcn/sd/cata/stackoverflow/entity/Question.java

```diff
@@ -0,0 +1,66 @@
+package ro.utcn.sd.cata.stackoverflow.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+//@Entity
+@AllArgsConstructor
+//@Table(name = "question")
+public class Question {
+    //@Id
+    //@GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+    //@Column(name = "author")
+    private Integer author;
+    //@Column(name = "title")
+    private String title;
+    //@Column(name = "text")
+    private String text;
+    //@Column(name = "creation_date")
+    private String creationDate;
```

Comment:
You should try to use the built-in classes for storing dates and times (LocalDate, LocalDateTime) instead of using a String. This is because these classes provide built-in operations for manipulating dates / times.